datacontract/export/great_expectations_converter.py

```diff
@@ -0,0 +1,135 @@
+        }
+    }
+
+
+def model_to_expectations(fields: Dict[str, Field]) -> list:
+    expectations = []
+    add_column_order_exp(fields, expectations)
+    for field_name, field in fields.items():
+        add_field_expectations(field_name, field, expectations)
+    return expectations
+
+
+def add_field_expectations(field_name, field: Field, expectations: []) -> dict:
+    if field.type is not None:
+        expectations.append(to_column_types_exp(field_name, field.type))
+    if field.unique is not None:
+        expectations.append(to_column_unique_exp(field_name))
+    if field.maxLength is not None or field.minLength is not None:
+        expectations.append(to_column_length_exp(field_name, field.minLength, field.maxLength))
+    if field.minimum is not None or field.maximum is not None:
+        expectations.append(to_column_min_max_exp(field_name, field.minimum, field.maximum))
+
+    # TODO: all constraints
+    return expectations
+
+
+def add_column_order_exp(fields: Dict[str, Field], expectations: []):
+    expectations.append({"expectation_type": "expect_table_columns_to_match_ordered_list",
+                         "kwargs": {
+                             "column_list": list(fields.keys())
+                         },
+                         "meta": {}
+                         })
+
+
+def to_column_types_exp(field_name, field_type) -> dict:
+    return {
+        "expectation_type": "expect_column_values_to_be_of_type",
+        "kwargs": {
+            "column": field_name,
+            "type_": field_type
+        },
+        "meta": {}
+    }
+
+
+def to_column_unique_exp(field_name) -> dict:
+    return {
+        "expectation_type": "expect_column_values_to_be_unique",
+        "kwargs": {
+            "column": field_name
+        },
+        "meta": {}
+    }
+
+
+def to_column_length_exp(field_name, min_length, max_length) -> dict:
+    return {
+        "expectation_type": "expect_column_value_lengths_to_be_between",
+        "kwargs": {
+            "column": field_name,
+            "min_value": min_length,
+            "max_value": max_length
+        },
+        "meta": {}
+    }
+
+
+def to_column_min_max_exp(field_name, minimum, maximum) -> dict:
+    return {
+        "expectation_type": "expect_column_values_to_be_between",
+        "kwargs": {
+            "column": field_name,
+            "min_value": minimum,
+            "max_value": maximum
+        },
+        "meta": {}
+    }
+
+
+def get_quality_checks(quality: Quality) -> dict:
+    if quality is None:
+        return {}
+    if quality.type is None:
+        return {}
+    if quality.type.lower() != "great-expectations":
+        return {}
+    if isinstance(quality.specification, str):
+        quality_specification = yaml.safe_load(quality.specification)
+    else:
+        quality_specification = quality.specification
+    return quality_specification
+
+
+def checks_to_expectations(quality_checks: {}, model_key: str) -> []:
```

Comment:
The typing conventions are not the consistent accross the module ([] vs list vs List). 
I would advice using List ([] is not conventional), and detail the types of returned objects every time